packages/desktop/electron-builder-config.js

```diff
@@ -116,6 +122,7 @@ const prereleaseNsisOptions = {
     nsis: {
         oneClick: false,
         allowToChangeInstallationDirectory: true,
+        debugLogging: true,
```

Comment:
I think this is not enough. You also need to add the following line to installer.nsh:

`LogSet on`